getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -58,16 +58,24 @@ Set up a target
 	 sudo emacs /etc/default/grub
 
    b) Change ``GRUB_DEFAULT=[n]`` to ``GRUB_DEFAULT=saved``.
-      
-   c) Add ``GRUB_DISABLE_SUBMENU=y`` to the end of the file and save it.
 
-      This step is necessary because submenus confuse ktest.
+   c) You must disable GRUB submenus because they confuse ktest:
+
+      .. code-block:: console
+
+	echo 'GRUB_DISABLE_SUBMENU=y' | sudo tee -a /etc/default/grub.d/disable-submenu.cfg
```

Comment:
the commit is saying that `GRUB_DISABLE_SUBMENU=true` is the documented setting and not  with `y`?

Reply:
- The documentation changed from `y` (unusual) to `true` (common)
- The _implementation changed from: "`y` only" to: "`y` OR `true`".

=> To be compatible with ALL grub versions, stick to `y`.